packages/suite-native-atoms/src/Modal/BottomModal.tsx

```diff
@@ -49,13 +54,18 @@ export const BottomModal = ({
     children,
     hasBackArrow = false,
 }: BottomModalProps) => {
-    const { applyStyle } = useNativeStyles();
+    const { applyStyle, utils } = useNativeStyles();
+    const insets = useSafeAreaInsets();
 
     const handleCloseModal = () => onVisibilityChange(false);
 
     return (
         <BottomModalContainer isVisible={isVisible} onClose={handleCloseModal}>
-            <Box style={applyStyle(modalWrapperStyle)}>
+            <Box
+                style={applyStyle(modalWrapperStyle, {
+                    insetBottom: Math.max(insets.bottom, utils.spacings.md),
```

Comment:
Please move Max to style, that's is place where this style related logic belongs and also it don't needs to be recalculated during every rerender